packages/@extension-kid/notification-service/NotificationService.ts

```diff
@@ -0,0 +1,44 @@
+import { injectable, inject, named } from "inversify";
+import { ExtensionProvider } from "@extension-kid/core";
+
+import { Notification } from "./Notification";
+
+const NotificationServiceExtensionType = Symbol("NotificationServiceExtension");
+const NotificationServiceType = Symbol("NotificationService");
+interface NotificationServiceExtensionInterface {
+  id: symbol;
+  supportedNotifications(): symbol[];
+  push(notification: Notification): void;
+}
+
+@injectable()
```

Comment:
Thought I was looking at Angular code for a second 🙂

Reply:
coffeescript ☕️?